istio-proxy/istio-proxy.spec

```diff
@@ -52,11 +51,16 @@ BuildRequires:  platform-python-devel
 
 Source0:        proxy-%{git_commit}.tar.gz
 
+ExclusiveArch:  x86_64
+
 %description
 The Istio Proxy is a microservice proxy that can be used on the client and server side, and forms a microservice mesh. The Proxy supports a large number of features.
 
 %prep
 %setup -q -n proxy-%{git_commit}
+%setup -q -D -T -n proxy-%{git_commit} -a 10
+pushd maistra/vendor/envoy
+popd
```

Comment:
Why this empty push?

Reply:
I'm not actually sure. Perhaps we should remove it from product too?